### PR TITLE
Only show logout link if you are logged in

### PR DESCRIPTION
**Pull request by Br3nda**
# What has changed and why?
the logout link was appearing before you log ing

# How to test this change
- [x] has automated tests
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: spec/features/logins.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+require 'rails_helper'
+
+RSpec.describe 'Logging in', type: :feature do
+  context 'anonymous' do
+    it "can't see it" do
+      visit '/admin'
+      expect(page).to have_text('Rates Rebate 2018/2019')
+      expect(page).to have_text('Log in')
+      expect(page).not_to have_text('Logout')
```
Comment (CaraHill) [inline spec/features/logins.rb]: Text should be `Log out`

File: spec/features/logins.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+  context 'signed in as dia' do
+    let(:user) { FactoryBot.create :admin_user }
+
+    before { login_as(user, scope: :user) }
+
+    it ' Can see rebate forms' do
+      visit '/admin'
+      expect(page).not_to have_text('Logout')
```
Comment (CaraHill) [inline spec/features/logins.rb]: As above. :)

Comment (Br3nda) [reply spec/features/logins.rb]: ooops. Thanks. And it should be `expect().to`, instead of not_to

Comment (CaraHill) [reply spec/features/logins.rb]: Of course! :)

**Pull request merged**